Modules/LearningSequence/classes/class.ilObjLearningSequenceAdminAccess.php

```diff
@@ -0,0 +1,8 @@
+<?php declare(strict_types=1);
+
+/**
+ * Class ilObjStudyProgrammeAdminAccess
+ */
+class ilObjLearningSequenceAdminAccess extends ilObjectAccess
```

Comment:
Is this enough? This somehow freaks me out. Don't we need to check any permissions?